kinto/core/views/heartbeat.py

```diff
@@ -17,6 +19,8 @@ def get_heartbeat(request):
     def heartbeat_check(name, func):
         status[name] = False
         status[name] = func(request)
+        # Rollback any open transaction (will release table locks etc.)
+        transaction.abort()
```

Comment:
Don't we want to do that at a upper level? I mean make sure the transaction have been canceled or committed each time a response have been sent?